packages/apps/composer-app/vite.config.ts

```diff
@@ -39,6 +39,7 @@ export default defineConfig({
     minify: process.env.DX_ENVIRONMENT === 'development' ? false : undefined,
     rollupOptions: {
       input: {
+        internal: resolve(__dirname, './internal.html'),
```

Comment:
could this be the default when DX_ENVIRONMENT === 'development'? is that what we'd want?

Reply:
right now we're not even sending any data from the dev environment, the api keys are only embedded into the production builds

Reply:
oh yeah, I had forgotten about that. Or to configure the Datadog key. Should we reconsider that now that we're using dev for our internal use?